src/Components/Pages/Calculator.js

```diff
@@ -1,13 +1,208 @@
-import PropTypes from 'prop-types';
+import PropTypes from 'prop-types'
 import React from 'react'
+import WASMWrapper from '../../Backend/WASMWrapper'
+
+import CalculatorSettings from '../CalculatorHelpers/CalculatorSettings'
+import CalculatorInput from '../CalculatorHelpers/CalculatorInput'
+import CalculatorOutput from '../CalculatorHelpers/CalculatorOutput'
+
+import * as convert from 'convert-units'
+
+const Units = {
+    Inputs: {
+        MeVPerNucleum: 'MeV/nucl',
+        //        Mev: 'Mev',
+    },
+    Outputs: {
+        SmallScale: 'keV/μm',
+        LargeScale: 'MeV/cm',
+        MassStoppingPower: 'Mev*g/cm^2'
```

Comment:
```suggestion
        MassStoppingPower: 'MeV*g/cm^2'
```

Reply:
But Electronvolt has the the SI symbol of 'eV'

Reply:
Yes, SI symbol for electronovolt is `eV`, if we add `M` prefix, then we get `MeV`